tests/test_simple_coverage_boost.py

```
@@ -1,47 +1,90 @@
+    def test_error_handling(self):
+        """Test error handling."""
+        client = TestClient(app)
+
+        # Test invalid endpoint
+        response = client.get("/invalid-endpoint")
+        assert response.status_code == 404
+
+    def test_lifespan_events(self):
+        """Test lifespan events."""
+        # This tests the lifespan context manager
+        with TestClient(app) as client:
```

Ruff (0.13.1)

Local variable `client` is assigned to but never used

Remove assignment to unused variable `client`

(F841)